.travis.yml

```diff
@@ -7,7 +7,10 @@ install:
    - wget http://d3kbcqa49mib13.cloudfront.net/spark-2.1.0-bin-hadoop2.6.tgz
    - tar xf spark-2.1.0-bin-hadoop2.6.tgz && mv spark-2.1.0-bin-hadoop2.6 spark
    - export SPARK_HOME=$(pwd)/spark
-   - export PYTHONPATH=$SPARK_HOME/python:$PYTHONPATH
 # command to run tests
-script: pytest
+script:
+   - export PYTHONPATH=$SPARK_HOME/python:$PYTHONPATH
+   - pytest
+   - unset PYTHONPATH
+   - pytest
```

Comment:
Just to justify those two 'pytest', we thought it would be nice to test the system in a pyspark-installed env (export PYTHONPATH) and also in a pyspark-missing env. That way we make sure that none of our tests are making strong assumptions about the actual environment.